pages/instructor-checklist.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: page-fullwidth
+show_meta: false
+title: "Instructor checklist"
+header:
+   image_fullwidth: "wood_plank.jpg"
+permalink: "/instructor-checklist/"
+---
+
+Each instructor is responsible for preparing to teach their section(s) of the workshop. Remember that our lessons are continuously evolving, so even if you have taught the content before, you will need to check the materials for changes.
+
+## Before the workshop:  
+- [ ] Coordinate with the lead instructor to decide who is teaching what, and in which order  
+- [ ] If your workshop is self-organized, meet with the lead instructor to be trained in [Data Carpentry’s teaching practices](https://swcarpentry.github.io/instructor-training/16-practices/)  
```

Comment:
"If your workshop is self-organized, and you are not a certified Data Carpentry instructor, meet with the lead instructor to be trained...."